README.md

```diff
@@ -1,98 +1,169 @@
-this project is trying to accomplish and why it matters. 
-Describe the problem(s) this project solves.
-Describe how this software can improve the lives of its audience.
+```shell
+$ pip install -r requirements.txt
+```
 
-Other things to include:
-
-  - **Technology stack**: Indicate the technological nature of the software, including primary programming language(s) and whether the software is intended as standalone or as a module in a framework or other ecosystem.
-  - **Status**:  Alpha, Beta, 1.1, etc. It's OK to write a sentence, too. The goal is to let interested people know where this project is at. This is also a good place to link to the [CHANGELOG](CHANGELOG.md).
-  - **Links to production or demo instances**
-  - Describe what sets this apart from related-projects. Linking to another doc or page is OK if this can't be expressed in a sentence or two.
-
-
-**Screenshot**: If the software has visual components, place a screenshot after the description; e.g.,
-
-![](https://raw.githubusercontent.com/cfpb/open-source-project-template/master/screenshot.png)
-
-
-## Dependencies
-
-Describe any dependencies that must be installed for this software to work. 
-This includes programming languages, databases or other storage mechanisms, build tools, frameworks, and so forth.
-If specific versions of other software are required, or known not to work, call that out.
+* [Flask](http://flask.pocoo.org/)
+* [Python icalendar](http://icalendar.readthedocs.org/en/latest/) 
+* [Python Dateutil](https://dateutil.readthedocs.org/en/latest/)
+* [Python mock (for the tests)](http://www.voidspace.org.uk/python/mock/)
 
 ## Installation
 
-Detailed instructions on how to install, configure, and get the project running.
-This should be frequently tested to ensure reliability. Alternatively, a link to
-another page is fine, but it's important that this works.
-
-## Configuration
-
-If the software is configurable, describe it in detail, either here or in other documentation to which you link.
-
-## Usage
-
-Show users how to use the software. 
-Be specific. 
-Use appropriate formatting when showing code snippets.
-
-## How to test the software
-
-If the software includes automated tests, detail how to run those tests.
+To clone and install `flask-eventics` locally in an existing Python 
+[`virtualenv`](https://virtualenv.pypa.io/en/latest/):
 
-## Known issues
+```shell
+$ git clone https://github.com/cfpb/flask-eventics
+$ pip install -e flask-eventics
+```
 
-Document any known significant shortcomings with the software.
+Note: this installs `flask-eventics` in 'editable' mode.
 
-## Getting help
+`flask-eventics` can be installed directly from Github:
 
-Instruct users how to get help with this software; this might include links to an issue tracker, wiki, mailing list, etc.
+```shell
+$ pip install git+https://github.com/cfpb/flask-eventics
+```
 
-**Example**
+Once installed, the `flask-eventics` blueprint simply needs to be
```

Comment:
Maybe this is a Python community convention that I'm not aware of, but our typical style is to refer to things like app names with no special formatting. Mind removing the backticks from all the references to flask-eventics that aren't in actual code?